app/views/application/references/alternative.njk

```diff
@@ -0,0 +1,64 @@
+{% extends "_form.njk" %}
+
+{% set type = applicationValue(["referees", id, "type"]) | lower %}
+{% set title = "Give details for your new referee" %}
```

Comment:
‘Details of new referee’ or ‘Add details of new referee’

Reply:
Going for "Add a new referee"